### PR TITLE
[Doc] add 2.3.7 rn

### PR DESCRIPTION
**Pull request by hellolilyliuyi**
## What type of PR is this：
- [ ] BugFix
- [ ] Feature
- [ ] Enhancement
- [ ] Refactor
- [ ] UT
- [x] Doc
- [ ] Tool

## Which issues of this PR fixes ：

Fixes #

## Checklist:

- [ ] I have added test cases for my bug fix or my new feature
- [ ] This pr will affect users' behaviors
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function

## Bugfix cherry-pick branch check:
- [ ] I have checked the version labels which the pr will be auto backported to target branch
  - [ ] 2.5
  - [ ] 2.4
  - [ ] 2.3
  - [ ] 2.2

### CODE REVIEW

**Review by EsoragotoSpirit — DISMISSED** [review]

File: docs/release_notes/release-2.3.md [hunk 1]
```diff
@@ -1,5 +1,18 @@
...
+## 2.3.7
+
+Release date: December 30, 2022
+
+### Bug fixes
+
+The following bugs are fixed:
+
+- A column named `a` in a StarRocks table is allowed to be NULL, but the column `a` in a view `t` that is created on that table is incorrectly set to be NOT NULL. [#15749](https://github.com/StarRocks/starrocks/pull/15749)
+- When data is loaded into StarRocks, a new tablet version is generated. But the FE may not yet be aware of the new tablet version. At that time, the execution plan for a query issued by FE still demands BEs to read the historical version of the tablet. If the garbage collection mechanism removes the historical version, the query cannot read the historical version and an error  "Not found: get_applied_rowsets(version xxxx) failed tablet:xxx #version:x [xxxxxxx]" is returned.  [#15726[(https://github.com/StarRocks/starrocks/pull/15726](https://github.com/StarRocks/starrocks/pull/15726))
```
Comment (evelynzhaojie) [inline docs/release_notes/release-2.3.md:12]: A new tablet version is generated when data is loaded into StarRocks. However, the FE may not yet detect the new tablet version and still require BEs to read the historical version of the tablet. If the garbage collection mechanism removes the historical version, the query cannot find the historical version and an error  "Not found: get_applied_rowsets(version xxxx) failed tablet:xxx #version:x [xxxxxxx]" is returned.

File: docs/release_notes/release-2.3.md [hunk 2]
```diff
@@ -1,5 +1,18 @@
...
 
+## 2.3.7
+
+Release date: December 30, 2022
+
+### Bug fixes
+
+The following bugs are fixed:
+
+- A column named `a` in a StarRocks table is allowed to be NULL, but the column `a` in a view `t` that is created on that table is incorrectly set to be NOT NULL. [#15749](https://github.com/StarRocks/starrocks/pull/15749)
```
Comment (evelynzhaojie) [inline docs/release_notes/release-2.3.md:11]: The column that is allowed to be NULL in a StarRocks table is incorrectly set to NOT NULL in a view created from that table.

File: docs/release_notes/release-2.3.md [hunk 3]
```diff
@@ -1,5 +1,18 @@
...
+- When data is loaded into StarRocks, a new tablet version is generated. But the FE may not yet be aware of the new tablet version. At that time, the execution plan for a query issued by FE still demands BEs to read the historical version of the tablet. If the garbage collection mechanism removes the historical version, the query cannot read the historical version and an error  "Not found: get_applied_rowsets(version xxxx) failed tablet:xxx #version:x [xxxxxxx]" is returned.  [#15726[(https://github.com/StarRocks/starrocks/pull/15726](https://github.com/StarRocks/starrocks/pull/15726))
+- When data is frequently loaded, FE takes up too much memory.  [#15377](https://github.com/StarRocks/starrocks/pull/15377)
+- For aggregation queries and multi-table JOIN queries, the statistics are not collected accurately and CROSS JOIN occurs in the execution plans, resulting in long query latency. [#15497](https://github.com/StarRocks/starrocks/pull/15497)
```
Comment (evelynzhaojie) [inline docs/release_notes/release-2.3.md:14]: aggregate queries

File: docs/release_notes/release-2.3.md [hunk 4]
```diff
@@ -1,5 +1,18 @@
...
+
+Release date: December 30, 2022
+
+### Bug fixes
+
+The following bugs are fixed:
+
+- A column named `a` in a StarRocks table is allowed to be NULL, but the column `a` in a view `t` that is created on that table is incorrectly set to be NOT NULL. [#15749](https://github.com/StarRocks/starrocks/pull/15749)
+- When data is loaded into StarRocks, a new tablet version is generated. But the FE may not yet be aware of the new tablet version. At that time, the execution plan for a query issued by FE still demands BEs to read the historical version of the tablet. If the garbage collection mechanism removes the historical version, the query cannot read the historical version and an error  "Not found: get_applied_rowsets(version xxxx) failed tablet:xxx #version:x [xxxxxxx]" is returned.  [#15726[(https://github.com/StarRocks/starrocks/pull/15726](https://github.com/StarRocks/starrocks/pull/15726))
+- When data is frequently loaded, FE takes up too much memory.  [#15377](https://github.com/StarRocks/starrocks/pull/15377)
```
Comment (evelynzhaojie) [inline docs/release_notes/release-2.3.md:13]: 主句放前面比较好

**Review by evelynzhaojie — DISMISSED** [review]

**Review by evelynzhaojie — APPROVED** [review]

**Review by EsoragotoSpirit — APPROVED** [review]

**Comment by hellolilyliuyi** [comment]
run starrocks_admit_test

**Pull request merged**